src/cluster/sharding.rst

```diff
@@ -666,25 +670,207 @@ cluster do not host any replicas for newly created databases, by giving
+
+.. code-block:: bash
+
+    $ curl -s $COUCH_URL:5984/db1/_shards | jq '.'
+    {
+      "shards": {
+        "00000000-7fffffff": [
+          "node1@127.0.0.1",
+          "node2@127.0.0.1",
+          "node3@127.0.0.1"
+        ],
+        "80000000-ffffffff": [
+          "node1@127.0.0.1",
+          "node2@127.0.0.1",
+          "node3@127.0.0.1"
+        ]
+      }
+    }
+
+In this example we'll split all the copies of the ``00000000-7fffffff`` range.
+The API allows a combination of parameters such as: splitting all
+the ranges on all the nodes, all the ranges on just one node, or one particular
+range on one particular node. These are specified via the ``db``,
+``node`` and ``range`` job parameters.
+
+To split all the copies of ``00000000-7fffffff`` we issue a request like this:
+
+.. code-block:: bash
+
+    $ curl -s -H "Content-type: application/json" -XPOST $COUCH_URL:5984/_reshard/jobs \
+      -d '{"type": "split", "db":"db1", "range":"00000000-7fffffff"}' | jq '.'
+    [
+      {
+        "ok": true,
+        "id": "001-ef512cfb502a1c6079fe17e9dfd5d6a2befcc694a146de468b1ba5339ba1d134",
+        "node": "node1@127.0.0.1",
+        "shard": "shards/00000000-7fffffff/db1.1554242778"
+      },
+      {
+        "ok": true,
+        "id": "001-cec63704a7b33c6da8263211db9a5c74a1cb585d1b1a24eb946483e2075739ca",
+        "node": "node2@127.0.0.1",
+        "shard": "shards/00000000-7fffffff/db1.1554242778"
+      },
+      {
+        "ok": true,
+        "id": "001-fc72090c006d9b059d4acd99e3be9bb73e986d60ca3edede3cb74cc01ccd1456",
+        "node": "node3@127.0.0.1",
+        "shard": "shards/00000000-7fffffff/db1.1554242778"
+      }
+    ]
+
+The request returned three jobs. One job for each of the three copies.
+
+To check progress of these jobs use ``GET /_reshard/jobs`` or ``GET
+/_reshard/jobs/{jobid}``.
+
+Eventually, these jobs should complete and the shard map should look like this:
+
+.. code-block:: bash
+
+    $ curl -s $COUCH_URL:5984/db1/_shards | jq '.'
+    {
+      "shards": {
+        "00000000-3fffffff": [
+          "node1@127.0.0.1",
+          "node2@127.0.0.1",
+          "node3@127.0.0.1"
+        ],
+        "40000000-7fffffff": [
+          "node1@127.0.0.1",
+          "node2@127.0.0.1",
+          "node3@127.0.0.1"
+        ],
+        "80000000-ffffffff": [
+          "node1@127.0.0.1",
+          "node2@127.0.0.1",
+          "node3@127.0.0.1"
+        ]
+      }
+    }
+
+.. _cluster/sharding/stop_resharding:
+
+Stopping Resharding Jobs
+------------------------
+
+Resharding at the cluster level could be stopped and then restarted. This can
+be helpful to allow external tools which manipulate the shard map to avoid
+interfering with resharding jobs. To stop all resharding jobs on a cluster
+issue a ``PUT`` to ``/_reshard/state`` endpoint with the ``"state": "stopped"``
+key and value. Can also specify an optional note or reason for stopping.
```

Comment:
```suggestion
key and value. You can also specify an optional note or reason for stopping.
```

Would it be worth making this a stronger suggestion? "You should also specify..."?